broken.js

```diff
@@ -0,0 +1 @@
+fodinfoidsifaf
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>'fodinfoidsifaf' is not defined.